src/components/Provision.js

```diff
@@ -32,26 +33,30 @@ const Fonts = createGlobalStyle`
 	}
 `;
 
-const Provision = ({ store, theme = {}, children }) => (
-	<Provider store={store}>
-		<ConnectedRouter history={history}>
-			<ThemeProvider theme={theme}>
-				<React.Fragment>
-					<Head />
-					<GlobalStyle />
-					<Authenticate>
+const Provision = ({ store, theme = {}, muiTheme, children }) => {
```

Comment:
No test for passing null or undefined for muiTheme....

Reply:
that kinda useless tests honestly speaking but, I will add it ok..

Reply:
Added test when no mui theme is passed to provision